Dev guide contextualization | Content release #2 | Go SDK (#2334)
temporalio/documentation: docs-src/go/project-setup-introduction.md

---
id: project-setup-introduction
title: Set up a new Temporal Application introduction
description:
sidebar_label: Project setup introduction
tags:
  - introduction
  - developer guide
  - project setup
---

:::competency Construct a new Temporal Application project

This section of the Temporal Go SDK developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application using Go.

By the end of this section you will know how to construct a new Temporal Application project.

:::

:::info Choose your development environment

There are three ways to follow this guide:

> - Use a local dev server

Author: Will link them up at min.

---
id: project-setup-introduction
title: Set up a new Temporal Application introduction
description:
sidebar_label: Project setup introduction
tags:
  - introduction
  - developer guide
  - project setup
---

This section covers how to use a terminal, a code editor, and a development Cluster to create a new Namespace, write a single Activity Workflow, run a Worker that talks to your development Cluster, run a Workflow using the CLI, add a testing framework, and view Workflows in the Web UI.

:::competency Construct a new Temporal Application project

This section of the Temporal Go SDK developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application using Go.

By the end of this section you will know how to construct a new Temporal Application project.

:::

:::info Choose your development environment

There are three ways to follow this guide:

- [Use a local dev server](/go/choose-dev-cluster#local-dev-server)
- [Use Temporal Cloud](/go/choose-dev-cluster#temporal-cloud)

> - [Use a Self-hosted environment such as Docker](/go/choose-dev-cluster#self-hosted-temporal-cluster)

Reviewer: ```suggestion
- [Use a self-hosted environment such as Docker](/go/choose-dev-cluster#self-hosted-temporal-cluster)
```

> Read more about [Which development Cluster you should choose](/go/choose-dev-cluster) below.

Reviewer: ```suggestion
Read more in the [Choose a development Cluster](/go/choose-dev-cluster) section on this page.
```
